src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -418,6 +422,9 @@
                 this.donationMatchingRule = info.donationMatchingRule;
                 this.donationDateRange = info.donationDateRange;
                 this.postProcessClass = info.postProcessClass;
+                this.processUsingScheduledJob = info.processUsingScheduledJob;
+                this.noMatchOnDate = (info.donationMatchingRule.indexOf("donation_date__c") < 0 && 
+                    info.donationMatchingRule.indexOf("npsp__donation_date__c") < 0);
```

Comment:
You would never reach ` info.donationMatchingRule.indexOf("npsp__donation_date__c") < 0`

Reply:
Oh wait - it's && so you would reach it, but ... you would never get a true in an unnamespaced org. Either way, not quite right.